packages/suite/src/actions/wallet/graphActions.ts

```diff
@@ -136,15 +137,16 @@ export const updateGraphData =
         const { graph } = getState().wallet;
 
         // TODO: default behaviour should be fetch only new data (since last timestamp)
-        let filteredAccounts: Account[] = accounts;
+        // exclude accounts with non-standard backend behaviour
```

Comment:
I'd suggest expading this comment with the reason why are coinjoin accounts excluded. Is it privacy? is it something else? :thinking:

Reply:
it is commented in isStandardBackendType utility itself:
```
// validate if account.backendType or NETWORK.customBackends are supported by TrezorConnect
```

but now i see that the comment is outdated :) customBackends does not belongs here anymore, i will change it